web/src/search_suggestion.ts

```diff
@@ -425,15 +461,19 @@ function get_topic_suggestions(last, terms) {
         // as the user makes their search more specific, subsequent calls to
         // this function will get more candidates from calling
         // stream_topic_history.get_recent_topic_names.
-        stream_topic_history_util.get_server_history(stream_sub.stream_id, () => {});
+        stream_topic_history_util.get_server_history(stream_sub.stream_id, () => {
+            // Do nothing, as we are only interested in triggering the request
+            // to fetch stream topic history.
```

Comment:
I think it would be best to just move the comment above into here, since it already explains why we don't use the results.